### PR TITLE
refactor!: remove interbtc maintenance fund, but transfer leftover to…

### PR DESCRIPTION
**Pull request by gregdhill**
… treasury

Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: interbtc-spec/docs/source/economics/incentives.rst [hunk 1]
```diff
@@ -76,14 +76,14 @@ interBTC interest process
 -------------------------
 
 Fees paid in interBTC (on Issue, Redeem, and Replace) are forwarded to a fee pool.
-The fee pool then distributes the interBTC fees to all Vaults, Maintainers, and Collators according to a configurable distribution, and, if implemented, depending on the SLA score.
+The fee pool then distributes the interBTC fees to all Vaults and Collators according to a configurable distribution, and, if implemented, depending on the SLA score.
```
Comment (sander2) [inline interbtc-spec/docs/source/economics/incentives.rst:79]: Interbtc 1.5 document mentions neither collateral, nor maintainers. Why did you remove maintainers but not collators?

Comment (gregdhill) [reply interbtc-spec/docs/source/economics/incentives.rst:79]: This was agreed to remove in the specification, I'm not sure what to do with the Collators - this is possibly related to the token model.

**Review by sander2 — APPROVED** [review]
